woocommerce/woocommerce
# Uganda Billing/Shipping address

### All Submissions:


### Changes proposed in this Pull Request:


#### Fix address format, billing/shipping address fields for Uganda Country.
> State becomes District
> Town / City becomes Town / Village
> Postal code, not required and hidden.

### How to test the changes in this Pull Request:

1. Add Items to Cart
2. Proceed to Checkout
3. In Country, choose Uganda
4. See fields Town / Village and District
5. Postal code field hides

### Other information:




### Changelog entry

> Fixed address format for Uganda.
> Fixed billing/shipping address for shoppers' billing/shipping address is for Uganda
